diff --git a/examples/CMakeLists.txt b/examples/CMakeLists.txt
index aa880ca..59d6153 100644
--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -1,4 +1,4 @@
-if(NOT DEFINED ENV{CIBUILDWHEEL})
+if(NOT DEFINED ENV{CIBUILDWHEEL} AND NOT DEFINED ENV{CONDA_BUILD})
 	# Can't compile binaries against libpython in CI
 	add_spt3g_executable(cppexample cppexample.cxx)
 	target_link_libraries(cppexample core)
